Code/Framework/AzCore/Platform/Common/WinAPI/AzCore/IO/SystemFile_WinAPI.cpp

```diff
@@ -177,22 +172,19 @@ namespace Platform
 {
     using FileHandleType = AZ::IO::SystemFile::FileHandleType;
 
-    void Seek(FileHandleType handle, const SystemFile* systemFile, SystemFile::SeekSizeType offset, SystemFile::SeekMode mode)
+    void Seek(FileHandleType handle, [[maybe_unused]] const SystemFile* systemFile, SystemFile::SeekSizeType offset, SystemFile::SeekMode mode)
```

Comment:
That's probably better done in a more holistic pass over the current file system. According to one Discord user the file stack isn't being outside of startup with it all going through AZ::IO::Streamer otherwise, so we're _this_ close to finally being able to clean the original file stack up without having to deal with a large fall out.

Reply:
will leave it for that pass, if you want, lets create another issue, I can remove it as part of another change.